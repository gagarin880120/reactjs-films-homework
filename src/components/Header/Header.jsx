import React from 'react';
import SearchFieldContainer from '../SearchField';
import styles from './Header.module.scss';

export default function Header() {
  return (
    <header className={styles.header}>
      <h1 className={styles.title}>
        Films
      </h1>
      <SearchFieldContainer />
    </header>
  );
    </header>
  )
}
