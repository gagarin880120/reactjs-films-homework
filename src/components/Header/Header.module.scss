.header {
  padding: 1% 0;
  display: flex;
  justify-content: space-between;
  font-family: Arial, Helvetica, sans-serif;
  color: #fff;
}

.title {
  font-size: 30px;
  text-transform: uppercase;
}
