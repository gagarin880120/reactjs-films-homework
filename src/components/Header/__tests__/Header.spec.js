import ShallowRenderer from 'react-test-renderer/shallow';
import React from 'react';
import Header from '../Header';

describe('Header component', () => {
  test('should render without crashing', () => {
    const renderer = new ShallowRenderer();
    renderer.render(<Header />);
    const result = renderer.getRenderOutput();
    expect(result).toMatchSnapshot();
  });
});

