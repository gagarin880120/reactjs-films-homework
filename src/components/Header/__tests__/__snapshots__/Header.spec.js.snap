// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header component should render without crashing 1`] = `
<header
  className="header"
>
  <Link
    to="/"
  >
    <h1
      className="title"
    >
      Films
    </h1>
  </Link>
  <Memo(Connect(SearchFieldContainer)) />
</header>
`;
